api/src/api/extracts_v1/extract_routes.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime, timedelta
+
+from flask import Response
+
+import src.adapters.db as db
+import src.adapters.db.flask_db as flask_db
+import src.api.extracts_v1.extract_schema as extract_schema
+import src.api.response as response
+from src.api.extracts_v1.extract_blueprint import extract_blueprint
+from src.auth.api_key_auth import api_key_auth
+from src.services.extracts_v1.get_extracts import ExtractListParams, get_extracts
+
+examples = {
+    "example1": {
+        "summary": "No filters",
+        "value": {
+            "pagination": {
+                "order_by": "created_at",
+                "page_offset": 1,
+                "page_size": 25,
+                "sort_direction": "ascending",
+            },
+        },
+    },
+}
+
+
+@extract_blueprint.post("/extracts")
+@extract_blueprint.input(
+    extract_schema.ExtractMetadataRequestSchema,
+    arg_name="raw_list_params",
+    examples=examples,
+)
+@extract_blueprint.output(extract_schema.ExtractMetadataListResponseSchema)
+@extract_blueprint.auth_required(api_key_auth)
+@flask_db.with_db_session()
+def extract_metadata_get(
+    db_session: db.Session, raw_list_params: dict
+) -> response.ApiResponse | Response:
```

Comment:
Should be able to do just `response.ApiResponse` - the search endpoint has `Response` as well because it can return a csv file

Reply:
Makes sense!